pkg/ruler/mapper.go

```diff
@@ -86,14 +85,15 @@ func (m *mapper) MapRules(user string, ruleConfigs map[string][]legacy_rulefmt.R
 	return anyUpdated, filenames, nil
 }
 
-func (m *mapper) writeRuleGroupsIfNewer(groups []legacy_rulefmt.RuleGroup, filename string) (bool, error) {
+func (m *mapper) writeRuleGroupsIfNewer(groups []rulefmt.RuleGroup, filename string) (bool, error) {
 	sort.Slice(groups, func(i, j int) bool {
 		return groups[i].Name > groups[j].Name
 	})
 
-	rgs := legacy_rulefmt.RuleGroups{Groups: groups}
+	rgs := rulefmt.RuleGroups{Groups: groups}
 
 	d, err := yaml.Marshal(&rgs)
+	level.Info(m.logger).Log("msg", "printing rulegroups before mapping to file", "rulegroups", d, "number of groups", len(groups))
```

Comment:
Looks quite an invasive log. Is your intention to keep it or was added while working on tests?